components/sidebar/sidebar_channel/sidebar_channel_link/sidebar_channel_link.tsx

```diff
@@ -150,6 +151,8 @@ export default class SidebarChannelLink extends React.PureComponent<Props, State
         } else if (event.shiftKey) {
             event.preventDefault();
             this.props.actions.multiSelectChannelTo(this.props.channel.id);
+        } else if (event.altKey) {
+            this.props.actions.markMostRecentPostInChannelAsUnread(this.props.channel.id);
```

Comment:
This branch also should `event.preventDefault()`. Otherwise, since the backing HTML element is an anchor tag, alt+click tries to bring up the option to save the content at the link as an html document (at least on my Ubuntu Linux / Brave browser setup).